scripts/publishCheck.js

```diff
@@ -0,0 +1,11 @@
+const packageJSON = require('../package.json');
```

Comment:
Make sure this scripts folder is not shipped with fdk final build

Reply:
No, this is not shipped with release build bundle. We explicitly mention under `files` key in package.json which all directories should be present.